api/applications.go

```diff
@@ -228,7 +228,7 @@ func getEnvironmentID(accountID int, applicationID int, environmentName string)
 }
 
 // ApplicationStatus returns a module's current status on Section's delivery platform
-func ApplicationStatus(accountID int, applicationID int) (as []AppStatus, err error) {
+func ApplicationStatus(accountID int, applicationID int, moduleName string) (as []AppStatus, err error) {
```

Comment:
I guess that you need a `ctx context.Context` as a first argument. This can't run forever so a reasonable timeout would be a good fit.

Reply:
Maybe outside of the scope of this PR.